src/components/graph/menus/modification-list-item.js

```diff
@@ -73,6 +73,8 @@ export const ModificationListItem = ({
             return modif.lineToAttachToId;
         } else if (modif.type === 'LINES_ATTACH_TO_SPLIT_LINES') {
             return modif.attachedLineId;
+        } else if (modif.type === 'DELETE_VOLTAGE_LEVEL_ON_LINE') {
+            return modif.attachedLineId;
```

Comment:
I dont know what needs to be displayed, but here we just have "Deleting a voltage level on a line" in the modification list,
because attachedLineId is not part of modif.
to be checked with PO ?